Project.toml

```diff
@@ -26,8 +26,15 @@ TZJData = "1"
 julia = "1.6"
 p7zip_jll = "17.4"
 
+[extensions]
+TimeZonesRecipesBaseExt = "RecipesBase"
+
 [extras]
+RecipesBase = "3cdcf5f2-1ef4-517c-9805-6587b60abb01"
```

Comment:
This is now required as on Julia 1.9 the `[weakdeps]` take precedence over `[deps]` so in order to test with this we need to now add `[extras]` right?

That explains the PkgBenchmark failure better too. You could work around that failure by setting the Julia version for the benchmark CI tests to 1.8 so that the `[weakdeps]` are ignored and `[deps]` are preferred. I'm fine with ignoring the benchmark tests for this PR so don't feel the need to go through this change.

Reply:
> This is now required as on Julia 1.9 the `[weakdeps]` take precedence over `[deps]` so in order to test with this we need to now add `[extras]` right?

Correct.

I would also prefer to just ignore the failure for this one PR so that benchmarking remains on the current release moving forward.